README.md

```diff
@@ -200,6 +200,14 @@ See [*selection*.data](#selection_data) for more.
 
 This method is not intended for concatenating arbitrary selections, however: if both this selection and the specified *other* selection have (non-null) elements at the same index, this selection’s element is returned in the merge and the *other* selection’s element is ignored.
 
+<a name="selection_selectChild" href="#selection_selectChild">#</a> <i>selection</i>.<b>selectChild</b>([<i>selector</i>]) [<>](https://github.com/d3/d3-selection/blob/master/src/selection/selectChild.js "Source")
+
+Returns a new selection with the (first) child of each element of the current selection matching the *selector* (if specified).
```

Comment:
As we do for other methods, we should document the forms that *selector* can take: function (and then explain how the function is invoked and what it is passed) or string.

Reply:
Done. I was surprised that .selectChild() would erase the child's __data__ property with the parent’s, but that's consistent with what .select() does.